.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 .env
-Makefile
\ No newline at end of file
+Makefile
```

Comment:
you don't need to have `Makefile` ignored. In fact you need to have it in CVS most of thimes. I suggest to use [gitignore.io](https://gitignore.io) for `.gitignore` template generation. 
E.g. I use [the following template](https://www.toptal.com/developers/gitignore/api/go,intellij+all,visualstudiocode) (with some minor adjustments) in most of my projects.

Reply:
fixed